frontend/src/components/Image/ImageDefault.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import {css} from 'aphrodite';
+import styles from './styles';
+
+export default class ImageDefault extends React.Component {
+  state = {
+    image_error: false,
+    image_src: this.props.missing_url
+  };
+  componentWillMount = () => {
+    // data:image/jpeg;base64,data
+    const {src} = this.props;
+    if( src) {
+      if( src.indexOf( '/') === -1){
+        fetch( `/api/v1/image/${src}`).then( response => {
+          response.blob().then( blob => {
+            const url = URL.createObjectURL(blob);
+            this.setState( {image_src: url});
+          });
+        });
+      } else {
+        this.setState( {image_src: src});
+      }
+    }
+  };
+  componentWillReceiveProps = (props) => {
+    if( props.src !== this.props.src){
+      this.setState( {image_error: false});
+    }
+  };
+  onImageError = (e) => {
+    console.log( "image error for url:", e.target.src);
+    this.setState( {image_error: true, image_src: this.props.missing_url});
+  };
+  render = () => {
+    const {missing_url} = this.props;
+    const {image_error, image_src} = this.state;
+    return (
+      <div>
+        {image_error?
```

Comment:
space before the ? please

Reply:
have since read a better syntax for these, IMHO, putting ? and : on new lines ... I like it, do you two? :
```
{ image_error
  ? "true option"
  : "false option"
```